tests/mmm/test_budget_optimizer.py

```diff
@@ -630,3 +630,275 @@ def test_mmm_optimize_budget_callback_parametrized(dummy_df, dummy_idata, callba
     assert hasattr(opt_result, "success")
     assert hasattr(opt_result, "x")
     assert hasattr(opt_result, "fun")
+
+
+@pytest.mark.parametrize(
```

Comment:
Do we test for correctness of the optimization result in this library? Probably too heavy of a lift if that's not been the practice. But it would be really nice to have a test that demonstrates the optimizer arrives at the correct result with and without flighting.

Reply:
I got an issue opened #1846